modules/tinymce/src/themes/silver/main/ts/ui/context/ContextUi.ts

```diff
@@ -85,7 +86,7 @@ const renderContextToolbar = (spec: { onEscape: () => Optional<boolean>; sink: A
               {
                 bar: oldContents,
                 // TODO: Not working
```

Comment:
That TODO still relevant?

Reply:
No idea, and didn't want to touch it without knowing what it meant.

Reply:
I suspect the `Not working` part is because it doesn't refocus when "unsliding" as it creates a new DOM node, so attempting to focus the old thing doesn't work. That's just a guess though :shrug:

Reply:
Discussed with the team. This has been like this for 2yrs and we've not had a bug report, so we're not sure what it was referring to or if it's even relevant. As such, we're just going to remove it.